projects/ion/src/lib/spinner/types.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @typedef {'primary' | 'secondary' | 'danger'} IonSpinnerColor - Spinner color.
+ * @typedef {'sm' | 'md' | 'lg'} IonSpinnerTextSize - Spinner text size.
+ */
+
+export type IonSpinnerTextSize = 'sm' | 'md' | 'lg';
+export type IonSpinnerColor = 'primary' | 'secondary' | 'danger';
+
+export interface IonSpinnerProps {
```

Comment:
I believe that the property documentation is wrong, as it references another property.